Graphics/Archiver/CMakeLists.txt

```diff
@@ -132,7 +132,14 @@ if(GL_SUPPORTED OR GLES_SUPPORTED)
     if(PLATFORM_WIN32)
         target_link_libraries(Diligent-Archiver-static PRIVATE GLEW::glew opengl32.lib)
     elseif(PLATFORM_LINUX)
-        target_link_libraries(Diligent-Archiver-static PRIVATE GLEW::glew GL X11)
+        if(PLATFORM_FREEBSD)
+            find_package(OpenGL REQUIRED)
```

Comment:
In particular, when you do this, doesn't OpenGL export all the required lib directories?

Reply:
From my expurrience, using the short `-lpthread` form to link to `/usr/lib/libpthread.so` works, but that's also a base system component.  I've rarely seen it work for user-installed stuffs.